### PR TITLE
add new block `stars`

### PR DESCRIPTION
**Pull request by remi-dupre**
Add a new `stars` block which indicates if the POI is a starred restaurant or hotel and eventually gives its rating.

The format looks like follows:

```json
{
    "type": "stars",
    "ratings": [
        {"has_stars": "yes", "nb_stars": 4.0, "kind": "lodging"}
    ]
}
```

```json
{
    "type": "stars",
    "ratings": [
        {"has_stars": "yes", "nb_stars": null, "kind": "restaurant"}
    ]
}
```

### CODE REVIEW

File: idunn/blocks/stars.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                    if isinstance(stars, bool)
+                    else StarsAvailable.YES
+                ),
+                nb_stars=stars if isinstance(stars, float) else None,
+                kind=kind,
+            )
+            for kind, stars in [
+                (StarsKind.LODGING, place.get_lodging_stars()),
+                (StarsKind.RESTAURANT, place.get_restaurant_stars()),
+            ]
```
Comment (amatissart) [inline idunn/blocks/stars.py:49]: To help with the integration, perhaps we could keep only one of these ratings, depending on `place.get_class_name()` ?   
Then the client could just append the number of stars with the title of the poi class/subclass, without any redundant implementation on both sides, or risk of confusion.

cc @xem @bbecquet

Comment (remi-dupre) [reply idunn/blocks/stars.py:49]: What I had in mind is that erdapfel would blindly display all items of the list, but maybe we risk displaying irrelevant data?

File: idunn/blocks/stars.py [hunk 2]
```diff
@@ -20,6 +20,13 @@ class StarsKind(Enum):
     LODGING = "lodging"
     RESTAURANT = "restaurant"
 
+    def check_class(self, subclass: Optional[str]) -> bool:
+        return (self, subclass) in [
+            (self.LODGING, "lodging"),
+            (self.RESTAURANT, "restaurant"),
+            (self.RESTAURANT, "fast_food"),
+        ]
```
Comment (amatissart) [inline idunn/blocks/stars.py:28]: `subclass` should rather be named `class_name`

Comment (remi-dupre) [reply idunn/blocks/stars.py]: I just rewrote last commit as this is a very minor change.

**Review by amatissart — APPROVED** [review]

**Pull request merged**